modules/ROOT/partials/configuration/link_default_protocol.adoc

```diff
@@ -8,7 +8,13 @@ endif::[]
 
 This option allows you to set a default protocol for links when inserting/editing a link via the link dialog. The protocol will apply to any links where the protocol has not been specified and the prefix prompt has been accepted.
 
+ifeval::["{plugincode}" == "link"]
 NOTE: This option also applies to the xref:autolink.adoc[autolink] plugin.
+endif::[]
+
```

Comment:
```suggestion
```
Elsewhere if it's the same thing we keep the if statements together 🤷  Feel free to ignore.